examples/health-checks.yml.simple

```diff
@@ -0,0 +1,3 @@
+ports:
+  - 5500
+  - 6500
```

Comment:
Are these TCP checks? If so, what design were you thinking of for how to handle HTTP checks? And script based checks?

Reply:
Nice!

Minor suggestion: instead of just `service1`, you probably want `name: service1`. Same for the scripts: `script: /usr/local/bin/foo.sh`. That's because the syntax you're using there in yaml is a list of maps, and each item in the map needs a key.

Also, for consistency, `ports` should then become:

```yaml
tcp:
  - name: foo
    host: "localhost"
    port: 80
  - name: bar
    host: "localhost"
    port: 443
```

Reply:
I'll modify this and also put up a sample structure in the README and/or examples so it is clear...